imagedephi/main.py

```diff
@@ -153,35 +173,124 @@ def run(
     verbose,
     log_file,
     index,
+    command_file: Path,
+    file_list: Path,
 ):
     """Perform the redaction of images."""
     params = _check_parent_params(ctx, profile, override_rules, recursive, quiet, verbose, log_file)
     if params["verbose"] or params["quiet"] or params["log_file"]:
         set_logging_config(params["verbose"], params["quiet"], params["log_file"])
+    file_contents = {}
+    command_inputs: list[Path] = []
+    command_output: Path
+    cf_override_rules: Path | None = None
+    cf_profile: str = ""
+    # cf_rename will only be checked if rename is false so default should be false.
+    cf_rename: bool = False
+    cf_recursive: bool = False
+    cf_index: int = 1
+    if command_file:
+        if command_file.suffix not in [".yaml", ".yml"]:
+            click.echo(
+                "Command file does not match the expected format. \n"
```

Comment:
If we print the same message in four locations, we should define it somewhere and reuse it.  Also, as long as we are using click.echo, we shouldn't do line breaks ourselves, just do `click.echo(click.text_wrap(<text>))`.

Reply:
refactored in ce9d1869969f838b9d2adb48e249abbec0556c1a